gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/sql/execute/TablePrivilegeInfo.java

```diff
@@ -222,20 +227,48 @@ public void executeGrantRevoke( Activation activation,
 									List grantees)
 		throws StandardException
 	{
+		doExecuteGrantRevoke(activation, grant, grantees, columnBitSets, actionAllowed, true, td);
```

Comment:
Here in this method at the end we can send a message to the Lead node to invalidate all cached plans.